### PR TITLE
fix: remove the overly complicated support for orientation

### PR DESCRIPTION
**Pull request by doudou**
Use instead the simple string-based representation

### CODE REVIEW

File: src/Driver.hpp [hunk 1]
```diff
@@ -141,6 +141,9 @@ namespace imu_aceinna_openimu {
         /** Configure the offset between rtk-heading and mag-heading */
         void writeRTKHeading2MagHeading(base::Angle const& angle);
 
+        /** Configure the offset between rtk-heading and mag-heading */
```
Comment (wvmcastro) [inline src/Driver.hpp:144]: bad doc

File: src/Parameter.hpp [hunk 2]
```diff
@@ -29,7 +28,7 @@ namespace imu_aceinna_openimu {
         {"periodic-packet-rate", 4, PARAM_INTEGER, nullptr},
         {"acceleration-filter", 5, PARAM_INTEGER, nullptr},
         {"angular-velocity-filter", 6, PARAM_INTEGER, nullptr},
-        {"orientation", 7, PARAM_ORIENTATION, nullptr},
+        {"orientation", 7, PARAM_STRING, nullptr},
```
Comment (wvmcastro) [inline src/Parameter.hpp:31]: Is this orientation regarding the magnetometer axis swap thing? A doc would be nice for new users

Comment (doudou) [reply src/Parameter.hpp:31]: Yes. I was thinking the same ... was not 100% sure it works in the first place.

**Comment by doudou** [comment]
Ping @wvmcastro

**Pull request merged**